src/components/ProjectDetails/ProjectDetailsContainer.js

```diff
@@ -58,6 +58,21 @@ const ProjectDetailsContainer = ( ): Node => {
     } )
   );
 
+  const { data: usersObservations } = useAuthenticatedQuery(
+    [( "searchObservations_2", id )],
```

Comment:
This has extraneous parens and could be more descriptive, e.g. `searchObservationsByUserInProject`

Reply:
Fixed